Feature/four color light tower

Antweeker opened the pull request:
Depends on https://github.com/ed-xj/qm/pull/5. We should merge that first.

Change around the buttons in the dashboard view.

Antweeker committed:
Move the ugly base64 blob into a served image

In the dashboardview template, move that ugly base64 into an image file
and serve it. The huge blob is a huge distraction to me.

Antweeker committed:
Changed button layout on DashboardView

Changed the button layout on dashboardview per the bullet point outline.

ed-xj commented on ui/web/tpl/DashboardView.html:
```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
                         </div>
                         <div style="padding-top:0px;padding-left:300px;">
-                            <img src="data:image/jpeg;base64,/9j/4R0DRXhpZgAATU0AKgAAAAgADAEAAAMAAAABBb0AAAEBAAMAAAABA4gAAAECAAMAAAADAAAAngEGAAMAAAABAAIAAAESAAMAAAABAAEAAAEVAAMAAAABAAMAAAEaAAUAAAABAAAApAEbAAUAAAABAAAArAEoAAMAAAABAAIAAAExAAIAAAAgAAAAtAEyAAIAAAAUAAAA1IdpAAQAAAABAAAA6AAAASAACAAIAAgAEk+AAAAnEAAST4AAACcQQWRvYmUgUGhvdG9zaG9wIENTNiAoTWFjaW50b3NoKQAyMDE0OjAzOjIyIDE0OjE3OjEwAAAEkAAABwAAAA ...
+                            <img src="img/machine-diagram.jpg"
```
i don't think we need to keep this diagram in the dashboard view anymore. it should be replaced with hardware(sensor) communication status(online/offline). remember this?

Antweeker replied:
ahh yeah, I didn't start on that bullet point.

ed-xj commented on ui/web/tpl/DashboardView.html:
```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
-                                <button type="button" class="btn btn-lg btn-default" disabled="disabled">Default</button>
+                                <button type="button" class="btn btn-lg btn-warning" disabled="disabled">Idle</button>
+                                <button type="button" class="btn btn-lg btn-success">Running</button>
+                                <button type="button" class="btn btn-lg btn-primary" disabled="disabled">Unused</button>
```
For the blue light, we need to have a button without label. so please remove the `Unused` label here.

Antweeker replied:
bootstrap button acting strangely with a button without text. It resizes it. have to investigate further.

ed-xj commented:
hey man, looking good overall, smooth sailing so far? minor change needed and after that. :shipit:

Antweeker committed:
Added space on unused button to keep height

Added the character "&nbsp;" to the unused button in order to unify the
height around all buttons.

Antweeker commented:
@ed-xj alright, I removed "Unused" and added an invisible character, good to go?

ed-xj commented:
let's do it. :shipit:
